tests/trainer/test_checkpoint.py

```diff
@@ -114,19 +114,22 @@ def checkpointing_trainer_hparams(composer_trainer_hparams: TrainerHparams) -> T
 def assert_checkpoints_equivalent(hparams_a: TrainerHparams, checkpoint_file_a: str, hparams_b: TrainerHparams,
                                   checkpoint_file_b: str) -> None:
 
+    def _load_checkpoint(checkpoint_dir: str, filename: str):
```

Comment:
Can you move this file outside to a top-level function? Not a fan of functions defined in functions...